src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -77,7 +143,7 @@ public void Dispose()
             _parametersCache.Dispose();
         }
 
-        private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appContext, bool useAppContext)
+        private async Task<bool> IsEnabledPrivateAsync<TContext>(string feature, TContext appContext, bool useAppContext, CancellationToken cancellationToken)
```

Comment:
Private is the access modifier, doesn't belong in the method name. Isn't `IsEnabled` by itself available?

Reply:
Ok, I wasn't sure if we wanted to keep async in the method name but I can change it to that

Reply:
IsEnabledAsync works